hatch_build.py

```diff
@@ -0,0 +1,35 @@
+"""Custom build hook for Hatch."""
+
+import pathlib
+from typing import Any
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class CustomBuilder(BuildHookInterface):
+    """Custom build hook for Hatch."""
+
+    PLUGIN_NAME = "custom"
+
+    def finalize(
+        self, version: str, build_data: dict[str, Any], artifact_path: str
+    ) -> None:
+        """Finalize the build process.
+
+        Args:
+            version: The version of the package.
+            build_data: The build data.
+            artifact_path: The path to the artifact.
+        """
+        if not (pathlib.Path(self.root) / "scripts").exists():
+            return
+
+        for file in pathlib.Path(self.root).rglob("**/*.pyi"):
+            file.unlink(missing_ok=True)
+
+        import subprocess
+
+        subprocess.run(
+            ["python", "-m", "reflex.utils.pyi_generator"],
```

Comment:
```suggestion
            import sys
            [sys.executable, "-m", "reflex.utils.pyi_generator"],
```

must use the same python executable running the script when chaining execution.

this fixes the build issue on windows